src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const COPY_OF_STATE = { ...state };
+  const ARRAY_OF_STATES = [];
```

Comment:
we should use CONSTANT_VALUES if there is permanent value between different function's calls, here you can use just stateCopy and resultStates